bin/setup

```diff
@@ -48,19 +48,23 @@ FileUtils.chdir APP_ROOT do
 
   unless File.exist?('.env')
     log "== Setup .env file from .env.example =="
-    cp '.env.example', '.env'
+    system("cp .env.example .env")
   end
 
   log "== Preparing database =="
   log "⚠️  If you use docker to run postgres, make sure your database is running ⚠️"
-  system! 'bin/rails db:setup'
+  system! 'bin/rails db:drop db:create db:migrate'
+  system! 'bin/rails db:seed'
```

Comment:
Motivation behind this change: https://www.loom.com/share/acbf8627436a4e8ea7785095ea97f1fc

Reply:
That's interesting, you shouldn't need to do `migrate` if the schema loads okay. Be interested to see what happens if we breakdown the `db:setup` step into smaller chunks to see whats going on.

Generally running `db:migrate` may take a longer time and thus that's why using schema load is more ideal. https://edgeguides.rubyonrails.org/active_record_migrations.html#what-are-schema-files-for-questionmark

I can test this out tomorrow to see which part of our schema is not working.